pull.py

```diff
@@ -57,8 +57,40 @@
         'FDLv1.2',
         'FDLv1.3',
     ],
+    'FreeArt': [ # any version
+        'LAL-1.2',
+        'LAL-1.3',
+    ],
     'FreeBSDDL': ['FreeBSD'],  # unify (multi-tag)
-    # FIXME: still working through this
+    'NPL': [ #any version
```

Comment:
The FSF wording for this isn't “any version”, it's “[versions 1.0 and 1.1][1]”.

[1]: https://www.gnu.org/licenses/license-list.html#NPL